integration_test/setup.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+. .env
+
+DOCKER_COMPOSE="docker compose"
```

Comment:
_select_docker_compose not necessary?

Reply:
docker-compose is the 'old' way and is not updated anymore. I would prefer to move to docker compose instead as the VM's we have at HESI also support the newer, plugin version of docker compose.